### ISSUE TITLE
Add integration with Linear

### ISSUE THREAD

**Issue by rhys-vdw** [toggl/track-extension#2137]
**Link to service:** https://linear.app/

**Is there a free account option?**: Yes

### Describe the solution you'd like

A "start timer" button similar to the GitHub integration that prefills the task name with the issue name and ID.

#### Additional context

We're using Linear now instead of GitHub issues.

**Comment by HartS** [comment]
I made this PR a while back: https://github.com/toggl/track-extension/pull/2070

**Comment by rhys-vdw** [comment]
Would really appreciate this one. 🙏

**Comment by TheSoloJake** [comment]
[User request](https://app.intercom.com/a/inbox/ayixs927/inbox/admin/2137598/conversation/106481309991244?view=List)

**Comment by catdevnull** [comment]
this was merged already, should close?

**Comment by rhys-vdw** [comment]
Yeah, it's technically there but it doesn't actually add a button to the issue view:

![Image](https://github.com/user-attachments/assets/f8158f06-9936-4b07-bb61-aac1e8cba85a)

It's only in the kanban/list view:

![Image](https://github.com/user-attachments/assets/77321077-ed3a-4c97-aded-da90c1bb6b24)

But yeah, I guess it's good. I still don't use it for that reason though since I always look at my issues before I start them.

**Comment by catdevnull** [comment]
yep, that's also my experience and hope that it gets fixed. it looks like a bug though, i believe the code for it is there

**Comment by matthewford** [comment]
This ideally needs to prefil the task/project, and be present on more views within linear.
Could use the project to map to the toggl project and the task id to map to the task. I'm syncing tasks via Zapier and have the ID in the task description.
